src/Overlay.tsx

```diff
@@ -15,6 +15,7 @@ type StyledOverlayProps = {
   maxHeight?: keyof Omit<typeof heightMap, 'auto' | 'initial'>
   visibility?: 'visible' | 'hidden'
   anchorSide?: AnchorSide
+  portalContainerName?: string
```

Comment:
We should only need to define this prop in `OverlayProps`.  The values here are specifically for what gets passed in to the styled component.  You should be able to remove this line without issues